components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.component.scss

```diff
@@ -0,0 +1,116 @@
+@import "~styles/variables";
+@import "~styles/mixins";
+
+:host {
+  display: block;
+  position: relative;
+  padding-left: 35px;
+}
+
+.dropdown-label {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  position: relative;
+  z-index: 2;
+  padding: 10px 0;
+  margin-left: auto;
+  cursor: pointer;
+  border: none;
+  background: transparent;
+  font-weight: 600;
+
+  > chef-icon {
+    font-size: 16px;
+    color: $chef-primary-dark;
+    margin-left: 8px;
+  }
+
+  .selection-label {
+    display: inline-block;
+    max-width: 150px;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  .selection-count {
+    display: inline-block;
+    width: 20px;
+    height: 20px;
+    font-size: 14px;
+    line-height: 20px;
+    text-align: center;
+    border-radius: 50%;
+    margin-left: 10px;
+    background: $chef-grey;
+
+    &.active {
+      background: $chef-primary-bright;
+      color: $chef-white;
+    }
+  }
+}
+
+.dropdown {
+  display: block;
+  position: absolute;
+  right: -10px;
+  max-width: 250px;
+  border: 1px solid $chef-grey;
+  font-size: 14px;
+  z-index: 3;
+
+  &::before {
+    display: block;
+    position: absolute;
+    top: -6px;
+    right: 12px;
+    width: 10px;
+    height: 10px;
+    background: $chef-white;
+    transform: rotate(45deg);
+    border-left: 1px solid $chef-grey;
+    border-top: 1px solid $chef-grey;
+    content: '';
+  }
```

Comment:
This is some heckin' crazy css. A comment might be a bit helpful to the less experienced :)

Reply:
This `::before` block is drawing and positioning the small white triangle in the top right corner of the dropdown overlay:

![](https://user-images.githubusercontent.com/479121/56755302-7636f780-674c-11e9-91ea-58bc6b523d8a.png)